src/app/app.module.ts

```diff
@@ -20,13 +17,12 @@ import { HeaderComponent } from './shared/header/header.component';
     ContactUsComponent,
     FooterComponent,
     NotFoundComponent,
-    HeaderComponent
   ],
   imports: [
     BrowserModule,
     IonicModule.forRoot(),
     AppRoutingModule,
-    HttpClientModule
+    SharedModule
```

Comment:
Importing a shared module in the root module for the header component won't work for the feature module.

The shared module should also be imported into the featured module.

Reply:
Please check now @farman8799